superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx

```diff
@@ -178,7 +115,13 @@ export default function ActivityTable({
       />
       <>
         {activityData[activeChild]?.length > 0 ? (
-          <ActivityContainer>{renderActivity()}</ActivityContainer>
+          <ActivityContainer>
+            <ActivityTableRow
```

Comment:
nit: Does it makes sense to call this ActivityCard ?

Reply:
I gave this name following the logic "table->row". If you look at ActivityTableRow it receives an array and returns several "cards" inside a Fragment.

I had thought of "ActivityList" (list of cards right?), but I chose to keep it as "ActivityTableRow" as there are some improvements to be made in the props and these components are very related...

Reply:
`TableRow` sounds like a row of a table, but it actually contains the whole table content for a selected tab. We should probably rename other `XxxTable` to simply `Recents`,  `Dashboards`, `Charts` and `SavedQueries` to be consistent with section titles, then the single table content can be named as `RecentsList`, `DashboardList`, etc.

Reply:
> `TableRow` sounds like a row of a table, but it actually contains the whole table content for a selected tab. We should probably rename other `XxxTable` to simply `Recents`, `Dashboards`, `Charts` and `SavedQueries` to be consistent with section titles, then the single table content can be named as `RecentsList`, `DashboardList`, etc.

I agree that the names are not good. But I think this is a task for another PR, the purpose of this was just to solve the problem of the keys and not to solve "all problems" related to these files.

Example: These files also need test files, the types need to be better defined, etc.
None of this is the scope of this PR, just as renaming the "ActivityTable" does not seem to me to be scope for this PR.

Reply:
@ktmud

Reply:
Agreed.